doc/articles/figma-plugin/designers/simple-login-page.md

```diff
@@ -0,0 +1,193 @@
+# Let’s Build a Simple Login Page!
+
+Follow this step-by-step guide, and you’ll learn how to build this page.
+
+![Simple Login Page](images/00_simple-login-page.png)
+
+**Step 1:** Setting the page template
+- How to Create a New Page?
+- What are the Layers in a Page Template?
+
+**Step 2:** Creating a form
+- How to Add a Text Box?
+- How to Change the Icon of a Text Box?
+- What Are the Layers in a Text Box?
+- How to Add a Password Box?
+- How to Change the Icon of a Password Box?
+- What Are the Layers in a Password Box?
+- How to Add a Button?
+
+**Step 3:** Adjusting the layout
+- How to Edit the Spacing Between the Elements?
+- How to Edit the Padding Around the Items?
+- How to Make the Content Centered Vertically?
+
```

Comment:
There's a prereqs and setup section missing here. It should link to the get started instructions in the [main Figma doc](https://github.com/unoplatform/uno/pull/7758/files).